build.gradle

```diff
@@ -0,0 +1,143 @@
+plugins {
+    id 'org.ajoberstar.grgit' version '4.1.0'
+    id 'com.github.johnrengelman.shadow' version '6.1.0'
+    id 'java'
```

Comment:
Why are you applying the `java` and `shadow` plugins to the root project? There isn't any Java source here.